egui/src/widgets/plotting/plot.rs

```diff
@@ -345,33 +124,6 @@ impl Plot {
         self
     }
 
-    /// If true, the x-bounds will be symmetrical, so that the x=0 zero line
-    /// is always in the center.
-    pub fn symmetrical_x_bounds(mut self, symmetrical_x_bounds: bool) -> Self {
```

Comment:
I agree that the new name (`center_x_axis`) is better, but keep the old function with an added `#[deprecated = "Renamed center_x_axis"]`